src/AWS.Logger.Core/AWSLoggerConfig.cs

```diff
@@ -35,6 +35,19 @@ public class AWSLoggerConfig : IAWSLoggerConfig
         /// </summary>
         public bool DisableLogGroupCreation { get; set; }
 
+        /// <summary>
+        /// Specifies the days to retain events in log groups which are created by this logger, if the one specified by <see cref="LogGroup"/> doesn't already exist
+        /// and <see cref="DisableLogGroupCreation"/> is not <c>true</c>. Requires logs:PutRetentionPolicy permission to apply the retention policy to
+        /// newly created log groups. The default value of <c>null</c> will not apply a retention policy to new log groups.
+        /// <para>
```

Comment:
In all the places this comment is copied can you add an additional note that errors setting the retention policy are treated as non-fatal and the application and logging will continue if the policy is failed to be set.

Reply:
Given the putting the policy will only be attempted once it might be worth making user errors from disallowed values for the retention policy to be fatal. All other errors e.g transient errors should remain non-fatal.

This will prevent the case of the instance responsible for creating the log group and putting the retention policy from only performing half its duty where users might expect idempotent behaviour

Reply:
I have implemented a change to make invalid retention policy values a fatal error in a separate PR that targets this PR's branch https://github.com/Will-Boulton/aws-logging-dotnet/pull/1. If you think it sounds like a good idea I will happily merge that PR into this one.

Reply:
I would say hold off at least for now. We generally don't put that type of validation on the client side because services change and add more possible values and then the library gets in the way until we get around to updating it.

Reply:
That makes sense, thanks